src/ast/passes/resource_analyser.h

```diff
@@ -47,6 +48,7 @@ class ResourceAnalyser : public Visitor {
   Probe *probe_;
 
   int next_map_id_ = 0;
+  size_t curr_tuple_depth_ = 0;
```

Comment:
Nit: why not use the same member var name as in codegen `tuple_depth_` ?

Reply:
No longer needed in current version